src/v2.rs

```diff
@@ -36,8 +37,7 @@ enum HeaderFrameKind {
 /// source maps.
 #[derive(Debug, Default)]
 pub struct EszipV2 {
-  modules: Arc<Mutex<HashMap<String, EszipV2Module>>>,
-  pub ordered_modules: Vec<String>,
```

Comment:
For reviewers: `ordered_modules` field was included in the public interface of this crate, but accodring to my search within Deno organization on GitHub, this field was only used by `src/examples/builder.rs`. So I think it's okay to remove it.